pkg/devfile/parser/context/context.go

```diff
@@ -45,13 +46,19 @@ type DevfileCtx struct {
 	// devfile json schema
 	jsonSchema string
 
-	//url path of the devfile
+	// url path of the devfile
 	url string
 
+	// token is a personal access token used with a private git repo URL
+	token string
+
+	// Git is an interface used for git urls
+	Git git.IGitUrl
```

Comment:
Do not expose the value. similar as other params. create `Set..()` and `Get...()` methods if you want to access from external.